project.clj

```diff
@@ -23,10 +23,11 @@
                  [com.cognitect/transit-clj "0.8.319"]      ;TODO move to separate http project
 
                  ;CLJS
-                 [org.clojure/clojurescript :scope "provided"]
+                 [thheller/shadow-cljs :scope "provided"]
                  [org.clojars.mmb90/cljs-cache "0.1.4"]     ;TODO move to separate cache project
                  [com.cognitect/transit-cljs "0.8.256"]     ;TODO move to separate http project
                  [com.lucasbradstreet/cljs-uuid-utils "1.0.2"]]
-  :profiles {:dev [:parent/dev {:dependencies [[io.jesi/customs "1.0.0-SNAPSHOT"] ;FIXME use managed version
+  :profiles {:dev [:parent/dev {:dependencies [[io.jesi/customs]
                                                [org.clojure/tools.namespace "0.3.1"]]}]}
-  :clean-targets ^{:protect false} ["target" ".shadow-cljs"])
+  :clean-targets ^{:protect false} [".shadow-cljs" ".cljs_node_repl" "out" :target-path]
```

Comment:
Worth killing .clj-kondo?

Reply:
Nah, the cache seems to be quite good a dealing with changing deps and versions. Don't think it's worth killing it